build.ps1

```diff
@@ -270,7 +270,7 @@ process
         }
 
         # Loading Build Tasks defined in the .build/ folder (will override the ones imported above if same task name).
-        Get-ChildItem -Path '.build/' -Recurse -Include '*.ps1' -ErrorAction Ignore |
+        Get-ChildItem -Path '.build/' -Recurse -Include '*.build.ps1' -ErrorAction Ignore |
```

Comment:
As discussed, I'm just afraid that's a breaking change if it inadvertently make it to the template (say by copy/paste)...

Reply:
Done. Move to `Sampler/scripts` and that is copied to root of built module. I rather would have wanted `.build/scripts` but then I would need to change build.ps1 again. So this was the best I could come up with.